lib/kubot/setup.rb

```diff
@@ -0,0 +1,27 @@
+module Kubot
+  class Setup
```

Comment:
That's weird class name, I don't think it's reusable at all, so I don't see who point of having it at all, we can just move this code into the main.rb.